src/axom/sidre/core/DataStore.cpp

```diff
@@ -86,6 +86,8 @@ DataStore::DataStore()
   , m_buffer_coll(new BufferCollection())
   , m_attribute_coll(new AttributeCollection())
   , m_need_to_finalize_slic(false)
+  , m_conduit_error_occurred(false)
+  , m_conduit_errors("")
```

Comment:
```suggestion
  , m_conduit_errors()
```

The double quotes in the arg list are not needed since the default std::string ctor will be called.